.gitignore

```diff
@@ -0,0 +1,2 @@
+*.un~
+*.sw[a-z]
```

Comment:
`.sw?` is for Vim files. Adding a `.gitignore` for those files is out of scope for this PR.
I recommend you to instead have a global gitignore (see the core.excludesfile option of git).

What about `*.un~`?